packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -66,191 +70,194 @@ const getBackgroundColors = () => {
 
 const cfg = (options?: Options): Config => {
   const { disablePreflight = true } = options || {};
+  // make palette colors (foundation colors) defined as css vars and make components tokens inherit it
+  const theme = getTokens(cssVarsFoundation);
+  const componentTokens = getComponentLevelToken(theme);
 
   return {
     content: ["auto"],
-    presets: [orbitFoundationPreset],
+    presets: [orbitFoundationPreset(theme)],
     corePlugins: {
       preflight: disablePreflight ? false : undefined,
     },
     theme: {
+      variables: {
+        // paletteBlueNormal -> --palette-blue-normal
```

Comment:
This comment doesn't add much value to the code. It is, in fact, a comment of what the `kebabCase` function does, and not what this piece of code does. We either update it to the correct outputted format or we remove it from here.